### PR TITLE
Cypress/E2E: Potential fix for recently used emoji spec

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
- recently_used_emoji_spec is passing for me locally but noticed the issue on cypress run might have something to do with calling beforeEach on every test. Modified the test so that setup is only once, hopefully fixing the finicky test on cypress run
- Added `KNOWN ISSUE: MM-41538`

#### Ticket Link
NA

#### Screenshots
Screen Shot 2022-02-03 at 2 35 17 PM
Screen Shot 2022-02-03 at 2 37 51 PM

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — COMMENTED** [review]
Thank you for doing this @josephbaylon 🥂

**Review by Phrynobatrachus — APPROVED** [review]
ty for following up on this one

**Review by furqanmlk — COMMENTED** [review]
Thanks @josephbaylon , 1 test case MM-T4438 is failing in [e2e](https://automation-dashboard.vercel.app/cycles/49eb255b-5487-48f9-b662-cdf3f321c3db)

image

File: e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js [hunk 1]
```diff
@@ -218,7 +218,7 @@ describe('Mark as Unread', () => {
         verifyPostNextToNewMessageSeparator('post3');
     });
 
-    it('Should show cursor pointer when holding down alt', () => {
+    it('Should show cursor pointer when holding down alt -- KNOWN ISSUE: MM-41538', () => {
```
[the remark this answers is not in the document]
Comment (josephbaylon) [reply e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js:221]: @furqanmlk I'll demote it as part of weekly demotion/promotion on end of week (Friday for me).

[the remark this answers is not in the document]
Comment (josephbaylon) [reply e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js:221]: @furqanmlk As for `MM-T4438`, it looks like the potential fix doesn't work on cypress run. Will remove fix from PR if I can't get it to work.

[the remark this answers is not in the document]
Comment (josephbaylon) [reply e2e/cypress/integration/mark_as_unread/mark_as_unread_spec.js:221]: @furqanmlk I separated MM-T4438 to its own file for now and simplified the before blocks on each file (removed unnecessary variables). Please rerun e2e again once the build passes. Thanks!

Screen Shot 2022-02-03 at 2 35 17 PM
Screen Shot 2022-02-03 at 2 37 51 PM

**Comment by josephbaylon** [comment]
@furqanmlk Looks like the new solution fixed the issue -- https://mattermost-cypress-report.s3.amazonaws.com/146475-64d6d05-onprem-ent-webapp-pr-9744-mattermost/mm-ee-test64d6d05/mochawesome.html Don't mind the last failure which is unrelated

**Review by furqanmlk — APPROVED** [review]
Thanks @josephbaylon for fixing this issue 👍

**Pull request merged**